server/src/main/java/au/org/aodn/ogcapi/server/core/model/ContactModel.java

```diff
@@ -0,0 +1,26 @@
+package au.org.aodn.ogcapi.server.core.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+public class ContactModel {
+
+    // Not include all fields according to this: https://github.com/stac-extensions/contacts
+    // The types of some fields are also not aligned with the spec.
+    // Currently only include the fields that are in use.
+    // May need to add more fields / modify some fields in the future.
+    protected String name;
+    protected String organization;
+    protected String identifier;
+    protected String position;
+    protected List<String> emails;
+    protected List<AddressModel> addresses;
+    protected List<LinkModel> links;
+    protected String roles;
```

Comment:
roles is List<String> or just String?

Reply:
the [recommended roles](https://github.com/stac-extensions/contacts?tab=readme-ov-file#contact-object) is a string list, but current returning field from elasticsearch is just a string, for some reasons. So i put a comment here to show this: 
![image](https://github.com/aodn/ogcapi-java/assets/102270719/c5033e52-cd8e-4248-a128-4ca0c818e7f0)
I remember not only roles here, also some other fields have a different type from the recommended one. May need to change in the future when needed.

Reply:
as discuss, change the elasticsearch's index schema if possible , use `nested` type for holding array. Then run `all` endpoint to regenerate the new index with latest schema, because the new `all` endpoint is slow so use local geonetwork with just a few records instead of the edge one (declare in environment variable of the es-indexer)

Reply:
Also, in the scenario where changing the ES indexer to get the right schema may take more effort than expected, which should be delayed in a new ticket, and you still need to comply with the STAC definition, fake it until you make it. For example, roles in the ES index is still just a text type (not nested as you will have to do).

Please work around this on the OGC-API side, e.g., use Arrays.asList() to convert a single string value into an array with a single element.

The STAC schema definition should be complied with the highest priority; otherwise, in the future, when needed, there will be errors! Some fields in the STAC schema are meant to be strictly complied with, not just suggested, e.g., the bounding box.